### ISSUE TITLE
Issue occurred when hit CreateAnother after editing

### ISSUE THREAD

**Issue by mohamedsabil83** [laravel-filament/filament#1836]
### Package

filament/filament

### Package Version

v2.10.23

### Laravel Version

v9.3.1

### Livewire Version

v2.10.4

### PHP Version

PHP 8.1.3

### Bug description

After editing an existing record and starting to create a new one, when hit **Create** saved without issue. but, if hit **Create and Create another**, saved but show the following issue and the saved record appears after refreshing the page.

![Screenshot from 2022-03-08 23-31-40](https://user-images.githubusercontent.com/10126040/157329479-fb59959d-cd2e-411c-91a0-a6ea51b310f1.png)
![Screenshot from 2022-03-08 23-43-19](https://user-images.githubusercontent.com/10126040/157329502-681580eb-f352-4be4-9994-bc76975b7a90.png)

### Steps to reproduce

Create a table with two columns:
- name => string
- is_enabled => boolean

This is the form schema:
```php
public static function form(Form $form): Form
{
    return $form
        ->schema([
            Forms\Components\TextInput::make('name')
                ->label(__('Name'))
                ->disableAutocomplete()
                ->required()
                ->unique(ignorable: fn (?Model $record): ?Model => $record),
            Forms\Components\Toggle::make('is_enabled')
                ->label(__('Enabled / Disabled'))
                ->default(true)
                ->inline(false)
                ->hint(fn ($state) => $state ? '<span class="font-bold text-success-700">'.__('Enabled').'</span>' : '<span class="font-bold text-primary-700">'.__('Disabled').'</span>')
                ->reactive(),
        ])
    ;
}
```

**Comment by intrepidws** [comment]
Are you using the import statement for the Model class?

`use Illuminate\Database\Eloquent\Model;`

**Comment by mohamedsabil83** [comment]
Yes, I did. For now, I just hid the **Create Another** button until completing the project. then, I'll spend time checking the reason.

**Comment by danharrin** [comment]
Fixed by #2057
